examples/cassandra/README.md

```diff
@@ -300,47 +226,106 @@ spec:
 [Download example](cassandra-controller.yaml?raw=true)
 <!-- END MUNGE: EXAMPLE cassandra-controller.yaml -->
 
-Most of this replication controller definition is identical to the Cassandra pod
-definition above; it simply gives the replication controller a recipe to use
-when it creates new Cassandra pods.  The other differentiating parts are the
-`selector` attribute which contains the controller's selector query, and the
-`replicas` attribute which specifies the desired number of replicas, in this
-case 1.
+There are a few things to note in this description.
+
+The `selector` attribute contains the controller's selector query. It can be
+explicitly specified, or applied automatically from the labels in the pod
+template if not set, as is done here.
+
+The pod template's label, `app:cassandra`, matches to the selector in the
```

Comment:
nit: wordy

"...matches the Service selector from Step 1.  This is how pods created by this replication controller are picked up by the Service."